src/pages/Dashboard/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import InventoryImport from '../../components/InventoryImport';
+import {
+  Container,
+  CardBody,
+  CardTitle,
+  Card,
+  CardHeader,
+  Title,
+} from './styles';
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <Container>
+        <Title>
+          <strong>Painel de Conversão</strong>
```

Comment:
```suggestion
          Painel de Conversão
```
Da para remover este strong e colocar Title como font-weight: bold